ctapipe/image/tests/test_toy.py

```diff
@@ -58,7 +58,7 @@ def test_skewed():
     model = SkewedGaussian(
         x=x, y=y, width=width, length=length, psi=psi, skewness=skewness
     )
-    image, signal, _ = model.generate_image(geom, intensity=intensity, nsb_level_pe=5,)
+    _, _, _ = model.generate_image(geom, intensity=intensity, nsb_level_pe=5,)
```

Comment:
then just don't save anything in variables at all :wink:

Reply:
ah...yeh...